### PR TITLE
ADD NCR80 ALPS SKFL/FR

### PR DESCRIPTION
**Pull request by MaiTheSan**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/salane/ncr80alpsskfl/keyboard.json [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        "nkro": false,
+        "rgblight": false
+    },
+    "indicators": {
+        "num_lock": "GP0",
+        "caps_lock": "GP1",
+        "scroll_lock": "GP2"
+    },
+    "layouts": {
+        "LAYOUT": {
```
Comment (dunk2k) [inline keyboards/salane/ncr80alpsskfl/keyboard.json:33]:
```suggestion
        "LAYOUT_all": {
```

File: keyboards/salane/ncr80alpsskfl/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
```
Comment (dunk2k) [inline keyboards/salane/ncr80alpsskfl/keymaps/default/keymap.c:19]:
```suggestion
    [0] = LAYOUT_all(
```

File: keyboards/salane/ncr80alpsskfl/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        KC_ESC,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,
+
+        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,  KC_BSPC,     KC_INS,  KC_HOME, KC_PGUP,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,               KC_DEL,  KC_END,  KC_PGDN,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
+        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_LSFT, KC_SLSH,                        KC_UP,
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                                      KC_RGUI, KC_MENU, KC_RCTL,               KC_LEFT, KC_DOWN, KC_RGHT
+        )
+};
```
Comment (dunk2k) [inline keyboards/salane/ncr80alpsskfl/keymaps/default/keymap.c:28]:
```suggestion
};

```

**Review by dunk2k — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Review by waffle87 — APPROVED** [review]

**Comment by fauxpark** [comment]
@MaiTheSan This board is missing some layouts. Please fix this.
```
⚠ salane/ncr80alpsskfl: "LAYOUT_all" should be "LAYOUT" unless additional layouts are provided.
```

**Pull request merged**